lua/core/packer.lua

```diff
@@ -2,7 +2,7 @@ local M = {}
 
 M.bootstrap = function()
   local fn = vim.fn
-  local install_path = fn.stdpath "data" .. "/site/pack/packer/start/packer.nvim"
+  local install_path = fn.stdpath(vim.g.vim_version > 7 and "state" or "data") .. "/site/pack/packer/start/packer.nvim"
```

Comment:
Similarly here, `packer.nvim`, if you are using the git command/Use NvChad as a Packer Bootstrap, then packer will only be downloaded at stdpath("data"). No need for an extra check here